### ISSUE TITLE
Issue with Training data box_coordinates direction

### ISSUE THREAD

**Issue by compadrejavo** [wenwenyu/PICK-pytorch#56]
Greetings, I wanted to point a conflict about the data format of training and test data:

The readme says that the direction of the coordinates of the bounding boxes must be **clockwise**, but looking at the examples given in the X00016469623.tsv file of this repo, the coordinates are **counterclockwise**.

`1,83,41,331,41,331,78,83,78,TAN WOON YANN,other`
`1,109,171,330,171,330,191,109,191,MR D.I.Y. (M) SDN BHD,company`

![image](https://user-images.githubusercontent.com/16692399/98035582-80cd5500-1df7-11eb-8000-812bc1bf84dc.png)

**Comment by huliang2016** [comment]
whether the coordinates of the bounding boxes are clockwise or not, this line of code should be able to handle it correctly.

https://github.com/wenwenyu/PICK-pytorch/blob/master/data_utils/documents.py#L108

**Comment by phongnd3482** [comment]
Your coordinates directions are wrong. In image, the coordinates are as following
![image](https://user-images.githubusercontent.com/42857467/99624513-ff193080-2a60-11eb-87f7-93f38c0e9627.png)

**Comment by compadrejavo** [comment]
You are right, I messed up the axis, I'll close this issue
